conf/distro/rpb-qt5.conf

```diff
@@ -0,0 +1,8 @@
+require conf/distro/include/rpb.inc
+
+DISTRO_NAME = "Reference-Platform-Build-QT5"
+
+PACKAGECONFIG_append_pn-qtbase = " gles2 eglfs fontconfig kms"
```

Comment:
let's add examples as well in PACKAGECONFIG and add them into the image by default.